domain.yml

```diff
@@ -15,63 +37,95 @@ entities:
 
 responses:
   utter_greet:
-  - text: "Hi! It's time for your daily wellness check. Tracking healthy habits is a great way to measure your progress over time. Would you like to answer a few questions about your health?"
+  - text: Hi! It's time for your daily wellness check. Tracking healthy habits is a great way to measure your progress over time. Would you like to answer few questions about your health?
+
+  utter_cheer_up:
+  - text: 'Here is something to cheer you up:'
+    image: https://i.imgur.com/nGF1K8f.jpg
+
+  utter_did_that_help:
+  - text: Did that help you?
+
+  utter_happy:
+  - text: Great, carry on!
 
   utter_goodbye:
-  - text: "See you tomorrow. Bye!"
+  - text: Bye
 
   utter_iamabot:
-  - text: "I am a bot, powered by Rasa."
-  
-  utter_ask_confirm_exercise:
+  - text: I am just another bot trying to help you.
+
+  utter_ask_health_form_confirm_exercise:
   - text: Did you exercise yesterday? Don't sweat it if you didn't run a marathon - walks count!
-  utter_ask_exercise:
-  - text: What kind of exercise did you do 💪 ?
-  utter_ask_sleep:
-  - text: "How much sleep did you get 💤  last night?"
-  utter_ask_diet:
-  - text: "Did you stick to a healthy diet 🥦  yesterday?"
-  utter_ask_stress:
-  - text: "What is your stress level right now 🧘 ?"
-    buttons:
-      - title: "low"
-        payload: '/inform{"stress": "low"}'
-      - title: "medium"
-        payload: '/inform{"stress": "medium"}'
-      - title: "high"
-        payload: '/inform{"stress": "high"}'
-  utter_ask_goal:
-  - text: "Setting goals - even small ones - is a great way to focus your day. What do you want to accomplish today 🥇 ?"
-  utter_slots_values:
-  - text: "You've submitted the following answers:\n
+
+  utter_ask_health_form_sleep:
+  - text: How much sleep did you get 💤 last night?
+
+  utter_ask_health_form_exercise :
+  - text: What kind of exercise did you do?
+
+  utter_ask_health_form_diet:
+  - text: Did you stick to a healthy diet 🥦 yesterday?
+
+  utter_ask_health_form_stress:
```

Comment:
Looks like the buttons in the response got dropped. Was that intended?

Reply:
yes, I did not put buttons intentionally. But now I feel, buttons would make it easy for user to answer. Thanks for brining this up, adding buttons back :)

Reply:
I changed response to below
```yml
  utter_ask_health_form_stress:
  - text: Is your stress level low, medium or high 🧘🏼?
    buttons:
    - title: "Low"
      payload: '/inform{"stress": "low"}'
    - title: "Medium"
      payload: '/inform{"stress": "medium"}'
    - title: "High"
      payload: '/inform{"stress": "high"}'
```
We have `stress` defined in slots as well as entities.

I am facing an error with buttons. Python is trying to interpret `{..}` as placeholder for string interpolation. It shows error in console. Reading rasa docs, I understood that payload should be a Json like this -> `{"entity": "value"}`

<img width="1513" alt="Screenshot 2021-04-20 at 2 33 52 PM" src="https://user-images.githubusercontent.com/25281055/115369192-7e17be00-a1e5-11eb-8ba1-341752c31640.png">

Though, if we ignore this error, bot seems to work. What do you suggest here?

Reply:
It works without throwing error. I referred [this stackoverflow answer](https://stackoverflow.com/a/5466478/7475099).

```yml
  utter_ask_health_form_stress:
  - text: Is your stress level low, medium or high 🧘🏼?
    buttons:
    - title: "Low"
      payload: '/inform{{"stress": "low"}}'
    - title: "Medium"
      payload: '/inform{{"stress": "medium"}}'
    - title: "High"
      payload: '/inform{{"stress": "high"}}'
```

(added `{{...}}` double curly brackets.)

<img width="1434" alt="Screenshot 2021-04-20 at 2 55 55 PM" src="https://user-images.githubusercontent.com/25281055/115372444-84f40000-a1e8-11eb-9516-45d448998c63.png">